pal/src/tlsio_arduino.c

```diff
@@ -2,14 +2,22 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
this file seems to have a mix of tabs and spaces. could we clean all the tabs to four spaces?

Reply:
Done.